src/integrationTest/kotlin/com/sourcegraph/cody/AllSuites.kt

```diff
@@ -0,0 +1,37 @@
+package com.sourcegraph.cody
+
+import com.sourcegraph.cody.agent.CodyAgentService
+import com.sourcegraph.cody.edit.DocumentCodeTest
+import com.sourcegraph.cody.util.CodyIntegrationTextFixture
+import java.util.concurrent.TimeUnit
+import org.junit.AfterClass
+import org.junit.runner.RunWith
+import org.junit.runners.Suite
+
+// We need a single tearDown() method running after all tests are complete,
+// so agent can be closed gracefully and recordings can be saved properly.
+// Due to the limitations of JUnit 4 this can be done only using SuiteClasses and AfterClass.
```

Comment:
This is because we are using JB testing classes (e.g. `BasePlatformTestCase`) which are based on specific `junit` version, and `junit4.jar` is part of the `ideaIC` package.
We will upgrade to `junit5` when we will upgrade base IJ version.